roles/config-rh-sso/tasks/realm.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: "Set realm facts for realm: {{ item.name }}"
+  set_fact:
+    realm_id: "{{ item.id | default(item.name) }}"
+    realm_name: "{{ item.name }}"
+    realm_displayName: "{{ item.displayName | default(item.name) }}"
+    realm_enabled: "{{ item.enabled | default(true) }}"
+    realm_sslRequired: "{{ item.sslRequired | default('external') }}"
+    realm_registrationAllowed: "{{ item.registrationAllowed | default(true) }}"
+    realm_emailLogin: "{{ item.emailLogin | default(true) }}"
+    realm_dupsAllowed: "{{ item.dupsAllowed | default(false) }}"
+    realm_resetPwd: "{{ item.resetPwd | default(false) }}"
+    realm_editUname: "{{ item.editUname | default(false) }}"
+    realm_bruteForceProtected: "{{ item.bruteForceProtected | default(true) }}"
+
+- name: "Retrieve template for realm: {{ item.name }}"
+  set_fact:
+    realm: "{{ lookup('template', 'realm.json.j2') | to_json }}"
+
+- name: "Create Realm for service Red Hat Single Sign-On"
+  uri:
+    url: "http://10.9.51.153:8080/auth/admin/realms"
```

Comment:
Need to get the IP of the SSO instance either through var or dynamically  instead of hard-coding it in the role